modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientExperimentalCodegen.java

```diff
@@ -897,6 +897,9 @@ public String getTypeString(Schema p, String prefix, String suffix) {
         if (")".equals(suffix)) {
             fullSuffix = "," + suffix;
         }
+        if (ModelUtils.isAnyTypeSchema(p)) {
+            return prefix + "bool, date, datetime, dict, float, int, list, str, none_type" + suffix;
```

Comment:
@sebastien-rosset Shall we remove prefix, suffix from this line as well?

Reply:
done